app/assets/stylesheets/article-show.scss

```diff
@@ -886,26 +868,15 @@ article {
     position: fixed;
     bottom: 0;
     z-index: 22;
-    background: white;
-    border: 1px solid darken($light-medium-gray, 2%);
-    @include themeable(
-      border,
-      theme-container-border,
-      1px solid darken(
-        $light-medium-gray, 2%
-      )
-    );
-    @include themeable(
-      box-shadow,
-      theme-container-box-shadow,
-      $bold-shadow
-    );
     width: 100%;
-    @include themeable(
-      background,
-      theme-background,
-      $tan
-    );
+    @include themeable(background, theme-container-background, $tan);
+    @include themeable(border, theme-container-border, none);
+    @include themeable(box-shadow, theme-container-box-shadow, $bold-shadow);
+
+    @media screen and (min-width: 950px) {
+      border-radius: 3px 3px 0 0;
```

Comment:
small comment, you can maybe clean this up with just 3px 0;

Reply:
`border-radius: 3px 0;` is a shorthand for `border-radius: 3px 0 3px 0;` so it's different than what I have - I know, CSS is weird... :)

Reply:
haha i actually realised that after i posted it and deleted the comment, but clearly it did not delete!